public/app/features/dashboard/components/GenAI/GenAIDashboardChangesButton.tsx

```diff
@@ -55,13 +57,19 @@ function getMessages(dashboard: DashboardModel): Message[] {
       role: Role.system,
     },
     {
-      content: `Group the following diff under "User changes" as a bullet list: ${JSON.stringify(userChanges)}`,
+      content: `Summarize the following user changes diff under a basic "User changes" heading as a bulleted list:\n${userChanges}`,
       role: Role.system,
     },
     {
-      content: `Group the following diff under "Migration changes" as a bullet list: ${JSON.stringify(
-        migrationChanges
-      )}`,
+      content: `Be sure to only include substantial user changes, such as adding or removing panels, changing panel titles or descriptions, etc. Ignore other changes and do not include them in the summary. Do not include "User Changes" section if there are no actual user changes to report.`,
+      role: Role.system,
+    },
+    {
+      content: `Threshold step changes of value being removed and replaced with color "green" should always be ignored`,
+      role: Role.system,
+    },
+    {
+      content: `If the following migrations diff has substantial changes add the following sentence at the end "It includes some changes made by Grafana toupdate the dashboard to the latest valid schema version." otherwise add "". Ignore any threshold step changes, templating list changes, and any diff less than 10 lines long. \n This is the migrations diff:\n${migrationChanges}`,
```

Comment:
```suggestion
      content: `If the following migrations diff has substantial changes add the following sentence at the end "Some autogenerated changes are included to update the dashboard to the latest valid schema version." otherwise add "". Ignore any threshold step changes, templating list changes, and any diff less than 10 lines long. \n This is the migrations diff:\n${migrationChanges}`,
```